e2e/test/scenarios/native-filters/helpers/e2e-date-filter-helpers.js

```diff
@@ -10,10 +10,10 @@ export function setMonthAndYear({ month, year } = {}) {
 }
 
 export function setQuarterAndYear({ quarter, year } = {}) {
-  cy.findByText(currentYearString).click();
+  popover().findByText(currentYearString).click();
 
-  cy.findByText(year).click();
-  cy.findByText(quarter).click();
+  popover().last().findByText(year).click();
```

Comment:
This is popover over popover. If `year` equals `currentYearString`, then without `last()` we find the text in the first popover.